index.html

```diff
@@ -4,10 +4,49 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Personal Portfolio</title>
+    <title>Document</title>
     <link rel="stylesheet" href="style.css">
 </head>
-<body>
-    
+<body class="flex_column">
+    <header>
+        <nav class="flex">
+            <div class="logo flex_column">
+                <a href=""><img src="assets/celec.png" alt="logo"></a>
+                <p>My logo</h2>
+            </div>
```

Comment:
- [ ] For this case, the link used in the [Figma template design]( should be text instead of an image, since you have added the text logo that wasn't there before, ensure enclosing the text in the `a` tag and may be customized the text to your name since this is your portfolio. Optionally, consider removing the image logo to match the Figma design ☺️.

Reply:
I added my own logo as suggested by the previous reviewer